css/global.css

```diff
@@ -1,23 +1,12 @@
-/* "more unreads" button, enable desktop notifications banner, and mention badge */
-.p-channel_sidebar__banner--unreads, .p-workspace_banner__desktop-notifications, .c-mention_badge {
-  display: none !important;
+body {
+  overflow: scroll !important;
+  position: initial !important;
 }
 
-#activity {
+div[id^="Hardsell"] {
```

Comment:
For any div's with id's that start with "Hardsell", it will remove them from the DOM.